src/link/preview.js

```diff
@@ -8,7 +8,7 @@ import {
 import { useAnchor } from '@wordpress/rich-text';
 import { __ } from '@wordpress/i18n';
 import { getPreviewHtml } from 'wikipedia-preview';
-import { isTextNearTheEdge } from './utils';
+import { isTextNearTheEdge, isTouch } from './utils';
```

Comment:
in Wikipedia-Preview we have added `mobile` class name to the preview when it is mobile device, do you think we can use it instead of another isTouch util here?